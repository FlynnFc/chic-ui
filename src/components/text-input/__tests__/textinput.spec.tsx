import TextInput from '..';
import { render } from '../../../test/utils';

/* eslint-disable */
describe('TextInput Component', () => {
  it('Renders Correctly', () => {
    const TextInputGroup = render(
      <>
        <TextInput placeholder="Type something..." size="small" />
        <TextInput placeholder="Type something..." width="200px" size="large" />
        <TextInput placeholder="Error Erorrr!" width="400px" error />
        <TextInput placeholder="Disabled Input" width="400px" disabled />
      </>
    );
    expect(TextInputGroup).toMatchSnapshot();
  });
});
