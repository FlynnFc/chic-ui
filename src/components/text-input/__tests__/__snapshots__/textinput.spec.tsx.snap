// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`TextInput Component Renders Correctly 1`] = `
Array [
  <div
    className="sc-bdfBQB ejubtU"
    width="100%"
  >
    <input
      className="sc-gsTEea itIBF"
      disabled={false}
      placeholder="Type something..."
      readOnly={false}
      type="text"
      width="100%"
    />
    <div
      className="sc-dlfnuX hDRiyF"
    />
  </div>,
  <div
    className="sc-bdfBQB hSRKwF"
    width="200px"
  >
    <input
      className="sc-gsTEea llmnwW"
      disabled={false}
      placeholder="Type something..."
      readOnly={false}
      type="text"
      width="200px"
    />
    <div
      className="sc-dlfnuX eHLKRu"
    />
  </div>,
  <div
    className="sc-bdfBQB dBKsns"
    width="400px"
  >
    <input
      className="sc-gsTEea grXQha"
      disabled={false}
      placeholder="Error Erorrr!"
      readOnly={false}
      type="text"
      width="400px"
    />
    <div
      className="sc-dlfnuX fFYkVR"
    />
  </div>,
  <div
    className="sc-bdfBQB dBKsns"
    width="400px"
  >
    <input
      className="sc-gsTEea glruUk"
      disabled={true}
      placeholder="Disabled Input"
      readOnly={false}
      type="text"
      width="400px"
    />
    <div
      className="sc-dlfnuX fFYkVR"
    />
  </div>,
]
`;
